src/common/win32/errno.cc

```diff
@@ -413,12 +413,14 @@ __s32 wsae_to_errno(__s32 r)
   return wsae_to_errno_unsigned(abs(r)) * sign;
 }
 
-__u32 errno_to_ntstatus(__u32 r) {
+__u32 errno_to_ntstatus(__s32 r) {
```

Comment:
As of fd04c30296899c8b9a0962edd14d26a11bf7abd8 (which will be backported to Pacific), there are now CephFS errnos which should be converted from instead.

Reply:
Thanks for bringing it up. I'll add another function called ``cephfs_errno_to_ntsatus`` and use that instead.